pegasus/sites.v3/advocacy.code.org/views/footer.haml

```diff
@@ -0,0 +1 @@
+#footer
```

Comment:
does this get used?

Reply:
Yes, it gives us a plain purple footer for now.